node/risk-app/README.md

```diff
@@ -15,13 +15,51 @@
+| OS_MAPS_URL                 | OS map API                              |   yes     |             |                             |       |
+| OS_NAMES_URL                | OS names API                            |   yes     |             |                             |       |
+| OS_SEARCH_KEY               | OS search key                           |   yes     |             |                             |       |
+| OS_MAPS_KEY                 | OS map key                              |   yes     |             |                             |       |
+| HTTP_PROXY                  |                                         |    no     |             |                             |       |
+| RATE_LIMIT_ENABLED          | Enable limit on postcode requests       |    no     | false       |                             |       |
+| RATE_LIMIT_REQUESTS         | Amount of requests per user             |    no     |             |                             |       |
+| RATE_LIMIT_EXPIRES_IN       | Time user request limit expires         |    no     |             |                             |       |
+| RATE_LIMIT_WHITELIST        | Limit enabled user exceptions           |   yes     | []          |                             |       |
+| REDIS_CACHE_ENABLED         | Redis cache storing enabled             |    no     | false       |                             |       |
+| REDIS_CACHE_HOST            | Redis linked AWS server URI             |    no     |             |                             |       |
+| REDIS_CACHE_PORT            | Redis default port                      |    no     | 6379        |                             |       |
+| COOKIE_PASSWORD             | Password to prevent exposing cookie data |  yes    |             |                             |       |
+| FRIENDLY_CAPTCHA_ENABLED    | Friendly Captcha robot check enabled    |    no     | false       |                             |       |
+| FRIENDLY_CAPTCHA_SITE_KEY   | Key for site for Catcha check           |    no     |             |                             |       |
+| FRIENDLY_CAPTCHA_SECRET_KEY | Secret key for website captcha          |    no     |             |                             |       |
+| FRIENDLY_CAPTCHA_URL        | Friendly Captcha API                    |    no     |             |                             |       |
+| FRIENDLY_CAPTCHA_BYPASS     | Friendly Capctha bypass key             |    no     |             |                             |       |
+| SESSION_TIMEOUT             | Timeout for session cookies             |    no     | 10          |                             |       |
+| RISK_PAGE_FLAG              | Flag to allow view of updated risk pages |   no    | false       |                             |       |
+| CACHE_ENABLED               | If enabled uses Redis caching           |    no     | true        |                             |       |
```

Comment:
I think the description needs to clarify that it's not related to using the redis server. So 'If enabled will cache external API calls'